### PR TITLE
Create SignUp page with Google SignIn button

### LINKED ISSUE #12: [Client] [SignIn] Create Sign in page with Google sign-in button
Create a signin page for Web App

image

### PR DESCRIPTION
**Pull request by rajat-sr**
Resolves #12

### CODE REVIEW

**Comment by rajat-sr** [comment]
SignUp page with Google SignIn button
homepage

File: client/src/components/GoogleSignIn/GoogleSignIn.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import GoogleSignInButton from './../../assets/google-button/btn_google_signin.png';
+
+const GoogleSignIn = () => {
```
Comment (nimish-gupta) [inline client/src/components/GoogleSignIn/GoogleSignIn.js]: no need of curly braces.

File: client/src/containers/SignUp/SignUp.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import GoogleSignIn from './../../components/GoogleSignIn/GoogleSignIn';
+import classes from './SignUp.module.css';
+
+class SignUp extends Component {
```
Comment (nimish-gupta) [inline client/src/containers/SignUp/SignUp.js]: use stateless component.

File: client/src/components/GoogleSignIn/GoogleSignIn.test.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import GoogleSignInButton from './../../assets/google-button/btn_google_signin.png';
+import { shallow } from 'enzyme';
+
+import GoogleSignIn from './GoogleSignIn';
+
+describe('<GoogleSignIn />', () => {
+  let wrapper;
```
Comment (nimish-gupta) [inline client/src/components/GoogleSignIn/GoogleSignIn.test.js]: why let?

Comment (rajat-sr) [reply client/src/components/GoogleSignIn/GoogleSignIn.test.js]: Changes made.

File: client/src/containers/SignUp/SignUp.js (hunk 2 above)
Comment (rajat-sr) [reply client/src/containers/SignUp/SignUp.js]: Changes made.

File: client/src/components/GoogleSignIn/GoogleSignIn.js (hunk 1 above)
Comment (rajat-sr) [reply client/src/components/GoogleSignIn/GoogleSignIn.js]: Changes made.

**Pull request merged**